script.js

```diff
@@ -0,0 +1,123 @@
+// functions
+//function for creating a element
+function createNode(element) {
+    return document.createElement(element)
+}
+
+// appendchild function
+function append(parent, element) {
+    return parent.appendChild(element)
+}
+
+//function for topstory, only one page at a time
+function getTopStory(pageNumber) {
+    const topStoryUrl = `https://newsapi.org/v2/top-headlines?country=gb&pageSize=1&apiKey=287554a05efe4127bd911a0a216a7b64&page=${pageNumber}`;
+    fetch(topStoryUrl)
+        .then(response => {
+            return response.json()
+        })
+        .then(body => {
+            const topUsaNewsArticles = body.articles
+            return topUsaNewsArticles.map(article => {
+                topStoryDiv.innerHTML = "";
```

Comment:
Also, it would be good to see display logic extracted out to own function so it could re-used later if needed